docs/current/sdk/python/snippets/private-repositories/clone.py

```diff
@@ -0,0 +1,31 @@
+# """
+# Clone a Private Git Repository and print the content of the README.md file
+# """
+
+import anyio
+import dagger
+
+async def private_repo():
+    async with dagger.Connection() as client:
+
+        repo = (
+            client.
```

Comment:
I noticed that in @helderco examples, the `.` starts each line, whereas here it ends each line. Not sure what the standard convention is but we should be consistent in our code formatting.

Reply:
I think there's more a consensus on starting with the `.` in general, even in Go.